.travis.yml

```diff
@@ -17,31 +17,25 @@ matrix:
     fast_finish: true
     include:
           # Test with lowest dependencies
-        - php: 7.1
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak"
-        - php: 5.5
+        - php: 7.2
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak"
 
           # Test the latest stable release
-        - php: 5.5
-        - php: 5.6
-        - php: 7.0
-        - php: 7.1
-        - php: 7.2
+        - php: 7.3
           env: COVERAGE=true TEST_COMMAND="composer test-ci"
-        - php: 7.2
+        - php: 7.3
           env: DEPENDENCIES="nikic/php-parser:^3"
 
           # Force some major versions of Symfony
-        - php: 7.2
+        - php: 7.3
           env: DEPENDENCIES="dunglas/symfony-lock:^3"
-        - php: 7.2
-          env: DEPENDENCIES="dunglas/symfony-lock:^4"
-        - php: 5.6
-          env: DEPENDENCIES="twig/twig:^1.30"
+        - php: 7.3
+          env: DEPENDENCIES="dunglas/symfony-lock:^4" SYMFONY_DEPRECATIONS_HELPER="2"
+
+        - php: 7.3
 
           # Latest commit to master
-        - php: 7.2
+        - php: 7.4snapshot
```

Comment:
7.4 (without snapshot) is not available yet?

Reply:
I didn't find this information. :/
Looks like symfony is still using `7.4snapshot` too (https://github.com/symfony/symfony/blob/master/.travis.yml#L32).
Do you know where to find the full list of supported PHP versions?

Reply:
It is now

Reply:
Perfect, I made the change! :+1: